Description: mib2c perl error message
 Change the error message of mib2c when it cannot find the perl module to install
 by using apt-get rather than downloading source code from the upstream website.
Author: Craig Small <csmall@debian.org>
Origin: Debian
Bug-Debian: https://bugs.debian.org/990706
Reviewed-by: Craig Small <csmall@debian.org>
Last-Update: 2021-07-06
--- a/local/mib2c
+++ b/local/mib2c
@@ -23,10 +23,9 @@
 if (!$havesnmp) {
     print "
 ERROR: You don't have the SNMP perl module installed.  Please obtain
-this by getting the latest source release of the net-snmp toolkit from
-http://www.net-snmp.org/download/ .  Once you download the source and
-unpack it, the perl module is contained in the perl/SNMP directory.
-See the README file there for instructions.
+this by running the command:
+
+apt-get install libsnmp-perl
 
 ";
     exit;
